src/main/kotlin/coffeebot/commands/CoffeeWager.kt

```diff
@@ -5,12 +5,13 @@ import coffeebot.message.Valid
 import org.jetbrains.exposed.sql.ResultRow
 import java.lang.StringBuilder
 
+private val userRegex = Regex("\\w+")
 private val coffeeRegex = Regex("(a|one|two|three|[1-7])(?: cups?(?: of)?)?(?: coffees?)?")
 private val betRegex = Regex("!bet $coffeeRegex (?:to $coffeeRegex )?that (.+)")
 private val cancelRegex = Regex("!cancel ([0-9]+)")
 private val acceptRegex = Regex("!accept ([0-9]+)")
-private val adjudicateRegex = Regex("!adjudicate ([0-9]+) ([A-Za-z]+)")
-private val payRegex = Regex("!pay (\\w+) $coffeeRegex")
+private val payRegex = Regex("!pay ($userRegex) $coffeeRegex")
```

Comment:
Kind of unrelated, I'm actually curious whether this is actually able to reuse the compiled versions of `userRegex` and `coffeeRegex`... I suspect it's just splicing the string representation of the regex into the new regex string, then compiling the whole thing. If that's true, `userRegex` and `coffeeRegex` could just be strings.

Reply:
I think you're right that it's not using the compiled version, as we're doing string interpolation. In any case, isn't it better that userRegex/coffeeRegex are Regexs for type safety?

Reply:
I guess I don't really know what type safety this gives us, since these are never used as regexes, and Kotlin will cast Regexes to Strings silently with no issue (which is what we're doing here).